tools/postman-collections/am-lib-testing.postman_collection.json

```diff
@@ -69,7 +69,7 @@
         ],
         "body": {
           "mode": "raw",
-          "raw": "{\n    \"resourceId\": \"my-super-resource-id\",\n    \"accessorIds\": [\"UNIQUE-USER-ID\"],\n    \"accessType\": \"user\",\n    \"serviceName\": \"cmc\",\n    \"resourceType\": \"case\",\n    \"resourceName\": \"claim\",\n    \"attributePermissions\": {\n    \t\"/test2\": [\"CREATE\", \"READ\"]\n    },\n    \"securityClassification\": \"PUBLIC\"\n}"
+          "raw": "{\n    \"resourceId\": \"my-super-resource-id\",\n    \"accessorIds\": [\"UNIQUE-USER-ID\"],\n    \"accessType\": \"USER\",\n    \"serviceName\": \"cmc\",\n    \"resourceType\": \"case\",\n    \"resourceName\": \"claim\",\n    \"attributePermissions\": {\n    \t\"/test2\": [\"CREATE\", \"READ\"]\n    },\n    \"relationship\": \"Defendant\",\n    \"securityClassification\": \"PUBLIC\"\n}"
```

Comment:
Neither `Defendant` nor `Solicitor` roles exist. Add it to the startup script of the testing harness otherwise requests will fail without manual intervention.

Reply:
Please apply similar changes to Gatling bodies.

Reply:
👍